test/functional_test.js

```diff
@@ -446,13 +451,16 @@ function runOnPrefix(apiPrefix) {
 
       it("should return a list of call-url with some information & subject",
```

Comment:
Alexis asked for it to be retained: https://github.com/mozilla-services/loop-server/pull/337#discussion_r32159044